test/e2e/info_test.go

```diff
@@ -49,4 +53,26 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.IsJSONOutputValid()).To(BeTrue())
 	})
+	It("podman info rootless storage path", func() {
+		if !rootless.IsRootless() {
+			Skip("test rootless_storage_path for rootless")
+		}
+		SkipIfRemote()
+		configPath := filepath.Join(os.Getenv("HOME"), ".config", "containers", "storage.conf")
+		rootStoragePath := `"/tmp/$HOME/$USER/$UID"`
+		cmd := exec.Command("sed", "-i", "-e", "/rootless_storage_path/d", "-e", fmt.Sprintf("/\\[storage\\]/a rootless_storage_path = %s", rootStoragePath), configPath)
+		err := cmd.Run()
+		Expect(err).To(BeNil())
+
+		expect := filepath.Join("/tmp", os.Getenv("HOME"), os.Getenv("USER"), os.Getenv("UID"))
+
+		session := podmanTest.Podman([]string{"info", "--format", "{{.store.GraphRoot}}"})
+		session.WaitWithDefaultTimeout()
+		Expect(session.ExitCode()).To(Equal(0))
+		Expect(session.OutputToString).Should(Equal(expect))
```

Comment:
I think the check shouldbe.
Expect(session.OutputToString).To(ContainString(expect))

Reply:
looks like this is not related. the result <func() string>: 0x1376d40 doesn't contains or equaal to <string>: /tmp/home/some11410dude/some11410dude
any other way to fix?